rir/src/runtime/ArgsLazyData.h

```diff
@@ -26,6 +26,9 @@ typedef std::function<SEXP(void*)> LazyFunction;
 
 struct ArgsLazyData : public RirDataWrapper<ArgsLazyData, LAZY_ARGS_MAGIC> {
     ArgsLazyData() = delete;
+    ArgsLazyData(const ArgsLazyData&) = delete;
+    const ArgsLazyData operator=(const ArgsLazyData&) = delete;
```

Comment:
The build failure is from cppcheck:

```
'ArgsLazyData::operator=' should return 'ArgsLazyData &'.
```

Reply:
I can never remember. why doesn't this language have sane defaults :(((